eng/Version.Details.xml

```diff
@@ -74,6 +70,11 @@
       <Uri>https://github.com/dotnet/core-setup</Uri>
       <Sha>d34d1570bfdf669ed94b4c939b2f39a1650e2320</Sha>
     </Dependency>
+    <!-- Keep this dependency at the bottom of ProductDependencies, else it will be picked as the parent for CoherentParentDependencies -->
```

Comment:
Question: The `CoherentParentDependency` attributes point CoreFx dependencies to Microsoft.NETCore.App.Runtime.win-x64. Why does a pinned Microsoft.NETCore.App.Ref dependency matter?

Reply:
The theory is that CPD traverses through versions.details.xml files top-down, looking for the first dependency from the repo it's tracking backwards towards. So since EFCore depends on a core-setup package through Extensions, DARC looks through Extensions for a core-setup package to use for dependency calculation. When the first one it finds is the pinned 3.0.0 one, it uses that. This PR is a hack/workaround until we improve the CPD algorithm.